cmd/up.go

```diff
@@ -158,6 +159,22 @@ List of components that kubeapps up installs:
 			return err
 		}
 		clientset, err := kubernetes.NewForConfig(config)
+		tries := 0
+		for {
+			p, ok, err := allReady(clientset)
+			if err != nil {
+				return err
+			}
+			if ok {
+				break
+			}
+			fmt.Printf("Kubeapps is starting up %v percentage...\n", int32(p))
```

Comment:
you probably want some ANSI escape code magic to update the percentage on the same line rather than reprint several times. See https://github.com/gosuri/uilive for an easy way to implement this.

Reply:
thanks @prydonius Lemme look